spec/controllers/graphql/queries/freaks_spec.rb

```diff
@@ -13,11 +13,17 @@ module Graphql
     end
 
     before do
-      create(:freak, first_name: 'Ion')
-      create(:freak, first_name: 'Gheorghe')
+      create(:role, id: 1, name: 'Founder')
+      create(:freak, role_id: 1)
     end
 
-    it { is_expected.to match_response_for(query: :freaks, sample: :freaks) }
```

Comment:
Why delete this test? The behavior of the code did not change. The response still contains a role.